bpf/lib/mcast.h

```diff
@@ -341,6 +342,7 @@ static long __mcast_ep_delivery(__maybe_unused void *sub_map,
 				struct _mcast_ep_delivery_ctx *cb_ctx)
 {
 	int ret = 0;
+	__u32 tunnel_id = 2;
```

Comment:
nit: If this is `WORLD_ID`, it should be `WORLD_ID`, not `2` :sweat_smile: 
```suggestion
	__u32 tunnel_id = WORLD_ID;
```

Reply:
Ack, this was there in the original PR too, but I guess it does not hurt to update in this diff.